### PR TITLE
Bump to 5.26.1

### PR DESCRIPTION
**Pull request by RichardIrons-neo4j**

### CODE REVIEW

File: Neo4j.Driver/common.props [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <Project>
     <PropertyGroup>
         <Copyright>Copyright 2002-2023</Copyright>
```
Comment (robsdedude) [inline Neo4j.Driver/common.props:3]: Unrelated: should this also be bumped at some point?
If so, maybe in a separate PR.

File: Neo4j.Driver/common.props
Comment (robsdedude) [inline Neo4j.Driver/common.props:1]: Why are we jumping from 5.25.0 to 5.26.1? Where did 5.26.0 go?

File: Neo4j.Driver/common.props (hunk 1 above)
Comment (RichardIrons-neo4j) [reply Neo4j.Driver/common.props:3]: Yeah probably.

File: Neo4j.Driver/common.props
Comment (RichardIrons-neo4j) [reply Neo4j.Driver/common.props:1]: I forgot to merge the GQL errors PR before releasing 5.26.0, as well as doing this version bump. 😳

**Review by robsdedude — APPROVED** [review]
:bubbles:

**Pull request merged**